### DISCUSSION TITLE
[Bug] [Java] Some unknown issues in skywalking service prevent this agent from being tracked

### DISCUSSION THREAD

**Discussion by GHQiuJun** [apache/skywalking#8105] [Q&A]
### Search before asking
- [x] I had searched in the issues and found no similar issues.

### Apache SkyWalking Component
- Java Agent

### Project Env
- java version "10.0.2" 2018-07-17
- skywalking agent   6.6

### Description
- From a certain point in time, two of the applications cannot be tracked by the skywalking service,and restarting the server and the client cannot be resolved. 
- But we changed the agent name and re-registered, and the service was tracked normally.
- Because we suspect that it is caused by some historical data of the server, but we can't confirm it, please give me some suggestions, thank you.

### Related issues
- No response

### Are you willing to submit a PR?
- [x] Yes I am willing to submit a PR!

### Code of Conduct
- I agree to follow this project's Code of Conduct

**Comment by GHQiuJun** [comment]
Such as ：2021-11-09 11:42:28.180 [TID:[Ignored Trace]] [http-nio-8080-exec-4]

**Comment by wu-sheng** [comment]
I think we are not aware of any case about this. Service name is irrelevant with any feature.
